dashboard/test/ui/features/signingIn.feature

```diff
@@ -50,3 +50,19 @@ Scenario: Teacher sign in from studio.code.org
   Then I wait until I am on "http://studio.code.org/home"
   Then I wait to see ".user_menu"
   Then element ".user_menu span:first" has text "Hi Casey"
+
+Scenario: Join non-existent section from sign in page shows error
+  Given I am on "http://studio.code.org/users/sign_in/"
+  And I type "9999999999" into "#section_code"
+  And I click selector "#section_form .btn-primary"
+  Then I wait until I am on "http://studio.code.org/courses"
+  And I wait for 10 seconds
```

Comment:
Can we replace this timed wait for a wait for a specific element to be visible instead?

Reply:
Oops, that was for local testing only as I was developing the test. My bad, good catch!